common/wireaddr.c

```diff
@@ -56,6 +231,12 @@ char *fmt_wireaddr(const tal_t *ctx, const struct wireaddr *a)
 		if (!inet_ntop(AF_INET6, a->addr, addrstr, INET6_ADDRSTRLEN))
 			return "Unprintable-ipv6-address";
 		return tal_fmt(ctx, "[%s]:%u", addrstr, a->port);
+    case ADDR_TYPE_TOR_V2:
+    
+		return tal_fmt(ctx, "%s.onion:%u", b32_encode(addrstr, (u8 *)a->addr) , a->port);  
+	case ADDR_TYPE_TOR_V3:
+		return tal_fmt(ctx, "%s.onion:%u", b64_encode(a->addr,42) , a->port);
```

Comment:
Should it not be using `b32_encode`? https://gitweb.torproject.org/torspec.git/tree/proposals/224-rend-spec-ng.txt#n2003 seems to say that the new V3 addresses still use base32 (the logic being that it replaces the domain name, and domain names are traditionally case insensitive, and base64 would introduce case sensitivity).  I believe we do not need base64 at all.

Reply:
removed all base64 changed to base32